internal/dataplane/parser/translate_grpcroute.go

```diff
@@ -99,10 +99,55 @@ func generateKongRoutesFromGRPCRouteRule(grpcroute *gatewayv1alpha2.GRPCRoute, r
 			},
 		}
 
+		// Kong routes derived from a GRPCRoute use a path composed of the match's gRPC service and method
+		// If either the service or method is omitted, there is a default regex determined by the match type
+		// https://gateway-api.sigs.k8s.io/geps/gep-1016/#matcher-types describes the defaults
+		if match.Method != nil {
+			var method, service string
+			matchMethod := match.Method.Method
+			matchService := match.Method.Service
+			var matchType gatewayv1alpha2.GRPCMethodMatchType
+			if match.Method.Type == nil {
+				matchType = gatewayv1alpha2.GRPCMethodMatchExact
+			} else {
+				matchType = *match.Method.Type
+			}
+			serviceMap := map[gatewayv1alpha2.GRPCMethodMatchType]string{
```

Comment:
If the values inside `serviceMap` and `methodMap` are not changed, could you define them as common variables, or pseudo-constants? 
Also, it will be clearer if we have some comments to tell what `""` (empty string) and `".+"` means in paths of kong routes.

Reply:
They don't have any special meaning in Kong routes, they're just regexes--the comment about the GEP section is where they come from.

Moved the maps into a static return in a helper function.